src/CrystallineSociety/Client/Shared/Pages/HomePage.razor.scss

```diff
@@ -2,103 +2,11 @@
 @import '../Styles/abstracts/_colors.scss';
 @import '../Styles/abstracts/_functions.scss';
 
-.page-container {
-    display: flex;
-    flex-flow: column nowrap;
-    justify-content: flex-start;
-    align-items: center;
-    width: 100%;
-    padding: 0 rem(16px);
-    box-sizing: border-box;
-}
-
-.page-section {
+.home-actions {
     width: 100%;
-}
-
-.hero-section {
     display: flex;
```

Comment:
For display and other attributes we have class name in Bootstrap like `d-flex`
@Marirood

Reply:
done.